main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        players_data = json.load(players)
+        for player, data in players_data.items():
+            nickname = player
+            email = data["email"]
+            bio = data["bio"]
+            race_name = data["race"]["name"]
+            race_description = data["race"]["description"]
+
+            race_info = Race.objects.get_or_create(
+                name=race_name,
+                description=race_description
+            )
+            print(race_info)
+            guild_data = data["guild"]
+            if guild_data is not None:
+                guild_name = guild_data["name"]
```

Comment:
https://github.com/mate-academy/py-game-models/blob/master/checklist.md#:~:text=1)%20Do%20not%20overload%20the%20context%20manager.

Reply:
Will it be a  good practice if I implement,for example, a separate  fucnction  in main.py  which will return a loaded data from json file and function main()  will add this data  to db?

Reply:
main() will have the same realization but without overloading context manager